dbtmetabase/_exposures.py

```diff
@@ -126,7 +126,7 @@ def extract_exposures(
                         continue
 
                     entity = dashboard_entity
-                    cards = entity.get("ordered_cards", [])
+                    cards = entity.get("dashcards", [])
```

Comment:
```suggestion
                    cards = entity.get("dashcards", entity.get("ordered_cards", []))
```

This appears to be a change in [Metabase 48](https://www.metabase.com/releases/metabase-48) in January, so a lot of people may still be running older versions